suite-native/device-manager/src/components/WalletItem.tsx

```diff
@@ -69,7 +85,14 @@ export const WalletItem = ({ deviceState, onPress, isSelectable = true }: Wallet
                     />
                     <Text variant="callout">{walletNameLabel}</Text>
                 </HStack>
-                {isSelectable && <Radio value="" onPress={onPress} isChecked={isSelected} />}
+                <HStack alignItems="center" spacing={12}>
+                    <FiatAmountFormatter
```

Comment:
Higher numbers does not fit on the screen and Radio is then out of the box. We should handle that. Maybe by using ellipsis for wallet label if it's too long?

Reply:
ca1e203d68

Reply:
nice 👍